src/main/java/oss/fosslight/service/impl/VerificationServiceImpl.java

```diff
@@ -1180,7 +1184,9 @@ public void updateStatusWithConfirm(Project project, OssNotice ossNotice) throws
 		
 		if(CoConstDef.FLAG_YES.equals(project.getAllowDownloadSPDXRdfYn())) {
 			if(isEmpty(spdxSheetFileId)) {
-				spdxSheetFileId = ExcelDownLoadUtil.getExcelDownloadId("spdx", project.getPrjId(), EXPORT_TEMPLATE_PATH);
+				Map<String, String> data = new HashMap<>(); data.put("prjId", project.getPrjId());

```

Comment:
If it is changed to JAVA 11 later, it will be changed to the method below.

``` java
Map<String, String> data = Map.of("prjId", project.getPrjId());
```